lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -276,41 +283,29 @@ def self.invoke_inline_ruby(aem, obj, inputs)
     def self.run_method(cmd)
       require 'open4'
       rc = nil
-      final_stderr = []
-      threads = []
       method_pid = nil
       begin
         status = Open4.popen4(*cmd) do |pid, stdin, stdout, stderr|
           method_pid = pid
           yield stdin if block_given?
           stdin.close
-          threads << Thread.new do
-            stdout.each_line { |msg| $miq_ae_logger.info "Method STDOUT: #{msg.strip}" }
-          end
-          threads << Thread.new do
-            stderr.each_line do |msg|
-              msg = msg.chomp
-              final_stderr << msg
-              $miq_ae_logger.error "Method STDERR: #{msg}"
-            end
-          end
-          threads.each(&:join)
+          stdout.each_line { |msg| $miq_ae_logger.info "Method STDOUT: #{msg.strip}" }
```

Comment:
@bdunne 
What happens to the DRb session errors where would they go. It does happen that we get session establishment errors, there could also be reams of data depending on what happens a SEGV. I think its good to route the STDOUT/STDERR from the method but the threads that drain the stdout/stderr will still help in dire situations. We had a tough time debugging this issue at the customer and the basic problem was a WinRM gem error that had problems and was spewing things into STDERR which we didn't drain.